### PR TITLE
Updated Greeting Feature to use LocalStorage

### PR DESCRIPTION
**Pull request by zaynaib**
I changed the greeting feature so it will display `Hello, user name`.
It uses local storage. Whenever the users refreshers the page, exit the browser and come back to the application the name will still be displayed.

I also added a placeholder to `greeting_input`. So that the user will know how to use the app.
The user can change the name displayed anytime after pressing the enter key.

### CODE REVIEW

File: js/greeting/greetings.js [hunk 1]
```diff
@@ -1,12 +1,58 @@
+import {displayTime} from './../time.js';
+
+console.log(displayTime().hour);
```
Comment (RafaelDavisH) [inline js/greeting/greetings.js:3]: @zaynaib - Would you be needing the console log for a future revisit?

**Review by RafaelDavisH — APPROVED** [review]
@zaynaib beautifully use of the time script and turning greeting to a module.

File: js/greeting/greetings.js (hunk 1 above)
Comment (zaynaib) [reply js/greeting/greetings.js:3]: @RafaelDavisH  No I don't think I will be needing it.

**Review by zaynaib — COMMENTED** [review]
@RafaelDavisH  Thanks for the feedback.

File: index.html [hunk 2]
```diff
@@ -48,7 +50,7 @@
     </div>
```
Comment (RafaelDavisH) [inline index.html]: @zaynaib The PR actually showed some conflicts. It seems to be missing @ming-yong changes for Responsiveness. Perhaps, your local branch needed to be updated before being pushed for PR by doing the 'git pull origin development'.

**Review by RafaelDavisH — CHANGES_REQUESTED** [review]
@zaynaib it seems to be missing @Ming(ming-yong) changes to the index.html for responsiveness.
I can do the changes manually, but it might be easier if you carry your changes to a new feature branch, maybe v2.  Before making the feature branch do 'git pull origin development'.

**Review by myg-main — CHANGES_REQUESTED** [review]
As @RafaelDavisH said, I think you are using the old version code without the responsive feature. Besides that, the code looks good!

**Comment by zaynaib** [comment]
@RafaelDavisH  I think I made all the changes.

**Review by RafaelDavisH — APPROVED** [review]
Thank you @zaynaib

**Review by myg-main — APPROVED** [review]
Great job!

**Pull request merged**